content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -149,7 +149,7 @@ If parameter configuration is needed, the following is an example of a Validatin
 with parameter configuration.
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?